workflows/adding-a-platform-api-endpoint.md

```diff
@@ -90,13 +90,15 @@ Things you may wish to change:
 
 Make sure your choice of runtime agrees with your local Node/Ruby version:
 * If using Ruby:
-  - Select version `2.5.0` as that's [the max runtime supported in AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)
-  - Officially declare your choice of ruby version in your repo via `echo "2.5.0" > .ruby-version`
+  - Select version `3.3.0` as that's [the max runtime supported in AWS Lambda](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)
```

Comment:
3.2 is specified in the sample yaml

Reply:
Maybe instead of hardcoding specific run times, we can link to the [aws docs ](https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html#runtimes-supported) that outline the most recent supported run times.